ApplicationLibCode/Application/RiaPreferencesSystem.cpp

```diff
@@ -77,6 +77,11 @@ RiaPreferencesSystem::RiaPreferencesSystem()
     CAF_PDM_InitField( &m_showPdfExportDialog, "showPdfExportDialog", true, "Show PDF Export Dialog" );
     caf::PdmUiNativeCheckBoxEditor::configureFieldForEditor( &m_showPdfExportDialog );
 
+    CAF_PDM_InitField( &m_useCylindricalCoordinateConversion, "useCylindricalCoordinateConversion", true, "Use Cylindrical Coords" );
```

Comment:
Use same name for member? Remove "conversion" part of name?

Reply:
fixed